modules/cnv-installing-virtio-drivers-existing-windows.adoc

```diff
@@ -5,32 +5,32 @@
 [id="cnv-installing-virtio-drivers-existing-windows_{context}"]
 = Installing VirtIO drivers on an existing Windows virtual machine
 
-Install the VirtIO drivers from the attached SATA CD drive to an 
-existing Windows virtual machine. 
+Install the VirtIO drivers from the attached SATA CD drive to an
+existing Windows virtual machine.
 
 [NOTE]
 ====
-This procedure uses a generic approach to adding drivers to Windows. The process
- might differ slightly between versions of Windows. Refer to the documentation for
- the version of Windows that you are installing.
+This procedure uses a generic approach to adding drivers to Windows.
+The process might differ slightly between versions of Windows.
+Refer to the installation documentation for your version of Windows
+for specific installation steps.
 ====
 
 .Procedure
 
 . Start the virtual machine and connect to a graphical console.
 . Log in to a Windows user session.
 . Open *Device Manager* and expand *Other devices* to list any *Unknown device*.
-.. You might need to open the `Device Properties` to identify the unknown device. 
+.. You might need to open the `Device Properties` to identify the unknown device.
```

Comment:
> ! need to v
Avoid using “need to” to refer to a requirement that a product or procedure imposes. Use “must” or, in
constructions in which “must” is not idiomatically correct, use “have to” or “required to.”

Reply:
Good catch. Yeah, switched to the imperative "open the..."